Formula/cayley.rb

```diff
@@ -68,22 +80,26 @@ def plist; <<~EOS
         <array>
           <string>#{opt_bin}/cayley</string>
           <string>http</string>
-          <string>--config=#{etc}/cayley.conf</string>
+          <string>--config=#{etc}/#{CONFIGURATION_FILE}</string>
         </array>
         <key>RunAtLoad</key>
         <true/>
         <key>WorkingDirectory</key>
         <string>#{var}/cayley</string>
         <key>StandardErrorPath</key>
-        <string>#{var}/log/cayley.log</string>
+        <string>#{var}/log/#{LOG_FILE}</string>
         <key>StandardOutPath</key>
-        <string>#{var}/log/cayley.log</string>
+        <string>#{var}/log/#{LOG_FILE}</string>
       </dict>
     </plist>
   EOS
   end
 
   test do
```

Comment:
Please check other tests for the usage of `fork`, which is the preferred method to run a server in a homebrew test. You can check httpd for example.